docs/03 - Implementing the Anchor Server/Communication/Events Schema.yml

```diff
@@ -219,9 +232,13 @@ components:
     StellarId:
       type: object
       description:
-        StellarId's are objects that identify an on-chain entity. Both end users and SEP-31 senders can be identified
-        with StellarId's. Id's for SEP-31 senders will never include a `memo` or `memo_type`.
+        StellarId's are objects that identify a SEP entity. Both end-users and SEP-31 senders/receivers can be identified
+        with a StellarId. Id's for SEP-12 customers, and consequently SEP-31 senders/receivers, will never include a `memo`
+        or `memo_type`.
```

Comment:
This isn't true. SEP-12 customers, when registered by a SEP-31 sender, will always have memos. This is because the public key is owned by the SEP-31 sender, so we need memos to differentiate the customers.

Reply:
IDs would be enough, but IDs aren't generated until the anchor sees the account + memo used to determine whether the a customer is new. So in the platform, we could send the IDs for customers and just the Stellar account for SEP-31 senders. Wdyt?

Or, we could just send all the info we have. So ID, account, & memo for customers, and just Stellar account for SEP-31 senders.

Reply:
There other issue is that in the future, SEP-24 or SEP-6 non-custodial wallets will need to be identified using their home domain (since they use client attribution). So it is even more convoluted.

Reply:
> IDs would be enough, but IDs aren't generated until the anchor sees the account + memo used to determine whether the a customer is new. So in the platform, we could send the IDs for customers and just the Stellar account for SEP-31 senders. Wdyt?

That's a good idea. We should go with that ✅ 

> Or, we could just send all the info we have. So ID, account, & memo for customers, and just Stellar account for SEP-31 senders.

I think this is the most reliable option and I'm up for going with that, but it will require a more thorough change:
* We purposely decided not to store the customer data in the Platform, so the Anchor Backend can be the single source of truth for customer info.
* Nevertheless, if we want to link the customer account+memo to the SEP-31 transaction, we will need to store it in our database as soon as we create the SEP-31 tx. In order to access that information, we will need to update the `GET {callbackApi}/customer` response body to include not only the id, but also the account and memo.

--
> There other issue is that in the future, SEP-24 or SEP-6 non-custodial wallets will need to be identified using their home domain (since they use client attribution). So it is even more convoluted.

Gotcha. That can be handled when we start supporting those SEPs.

Reply:
Updated in https://github.com/stellar/java-stellar-anchor-sdk/pull/412/commits/4cc4d677e9541afc453f7fd39c6ea3c5fa702591.